src/testdrive/tests/cli.rs

```diff
@@ -74,6 +86,22 @@ fn test_cmd_arg_missing_value() {
     );
 }
 
+#[test]
+fn test_cmd_arg_ci_output() {
+    cmd()
+        .arg("--ci-output")
+        .write_stdin("$ cmd badarg")
+        .assert()
+        .failure()
+        .stderr(
+            r#"^^^ +++ <stdin>:1:7: error: command argument is not in required key=value format
```

Comment:
Ohh, I see. Does Buildkite interpret this correctly? The way we do this in the shell scripts is we put the `^^^ +++` on its own line:

```
^^^ +++
error: some error occurred
```

Reply:
yup! check the error I linked to above.

Reply:
or just here: https://buildkite.com/materialize/tests/builds/13452#30729be6-3633-4a96-bff8-bc84c98e1220

Reply:
oh actually I guess that overwrites the section heading, which is not the same.

Reply:
which means I am changing it, the current output is weird.

Reply:
turns out I was wrong? buildkite only did it correctly for that first attempt, other ones ([example](https://buildkite.com/materialize/tests/builds/13466#5e97923e-8ac9-482a-b482-7a487b4a2c82)) did not handle this correctly. 🤷🏻‍♂️